Makefile

```
@@ -123,6 +124,12 @@ run-es: ## start elasticsearch backend
 	@$(COMPOSE_RUN) dockerize -wait tcp://elasticsearch:9200 -timeout 60s
 .PHONY: run-es
 
+run-graylog: ## start graylog backend
+	@$(COMPOSE) up -d graylog graylog_elasticsearch graylog_mongo
```

Compose dependency management will do it for you.

```suggestion
	@$(COMPOSE) up -d graylog
```